### PR TITLE
wizard test schema and tests

### PR DESCRIPTION
**Pull request by dot2dotseurat**
Brief comments on the purpose of your changes:
This purpose of this PR is to publish a github workflow that runs on workflow dispatch and upon merge that will run the modfied dbt models and tests compared to the production run. 

This workflow is protected by a codeowners file and concurrent runs per branch are not allowed. the workflow times out at 60 minutes. 

*For Dune Engine V2*
I've checked that:
General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if adding a new model, I edited the alter table macro to display new database object (table or view) in UI explorer
* [ ] if adding a new materialized table, I edited the optimize table macro

Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Each line is a file pattern followed by one or more owners.
```
Comment (a-monteiro) [inline .github/CODEOWNERS:1]: We don't need this file unless we want to get tagged if someone tries to change this, it won't actually prevent changes.

File: .github/protected-workflows.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# "events" is a map between Github events and rules.
```
Comment (a-monteiro) [inline .github/protected-workflows.yml:1]: This file isn't required

File: .github/workflows/dbt_slim_ci.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    timeout-minutes: 60
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
+      - name: Add git_sha to schema
+        env:
+          GIT_SHA: ${{ github.ref }}
+        run: sed -ie 's!^\(\s*schema:\)[^.]*!\1 git_$GIT_SHA!' profiles.yml
```
Comment (a-monteiro) [inline .github/workflows/dbt_slim_ci.yml:26]:
```suggestion
        run: sed -ie 's!^\(\s*schema:\)[^.]*!\1 git_$GIT_SHA!' profiles.yml
```
        run: "/runner/change_schema.sh git_$GIT_SHA"

File: .github/workflows/dbt_slim_ci.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+jobs:
+  dbt-test:
+    runs-on: [ self-hosted, linux, spellbook ]
+    timeout-minutes: 60
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+
```
Comment (a-monteiro) [inline .github/workflows/dbt_slim_ci.yml:22]: - name: Set environment variables
```suggestion
      - name: Set environment variables
        run: |
          echo "GIT_SHA=${GITHUB_SHA::7}" >> $GITHUB_ENV
```
I'm not sure if this works 😅

**Review by dsalv — APPROVED** [review]

**Pull request merged**